stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
```

Comment:
BTW, in case you missed it - @mhahn used brackets in his comment, and so it hid part of it.  Hit edit on his comment here and you'll see it.